README.md

```diff
@@ -2,7 +2,7 @@ Cloudinary
 ==========
 The official [Cloudinary](https://cloudinary.com) SDK for Angular.
 
-Cloudinary's AngularJS (AKA Angular1) SDK can be found [here](https://github.com/cloudinary/cloudinary_angular/tree/angular-1.x)
+Cloudinary's Angular SDK can be found [here](https://github.com/cloudinary/cloudinary_angular/tree/angular-5.x)
```

Comment:
Aren't we already in this repo?
it seems kind of strange to redirect people to the same repo.

Maybe we can say we maintain difference branches for different versions? 
What do you think?

Reply:
sure. updated